geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/serial/SerialWANPropagation_PartitionedRegionDUnitTest.java

```diff
@@ -371,4 +379,107 @@ public void testPartitionedSerialPropagationWithParallelThreads() throws Excepti
     vm2.invoke(() -> WANTestBase.validateRegionSize(getTestMethodName() + "_PR", 1000));
     vm3.invoke(() -> WANTestBase.validateRegionSize(getTestMethodName() + "_PR", 1000));
   }
+
+  @Test
+  public void testPartitionedSerialPropagationWithGroupTransactionEventsAndMixOfEventsInAndNotInTransactions()
```

Comment:
All comments that apply to `ParallelWANPropagationDUnitTest` also apply to this test method, since the code in both tests is largely identical.